aptos-move/mvhashmap/src/versioned_group_data.rs

```diff
@@ -77,6 +77,30 @@ impl<T: Hash + Clone + Debug + Eq + Serialize, V: TransactionWrite> Default
 }
 
 impl<T: Hash + Clone + Debug + Eq + Serialize, V: TransactionWrite> VersionedGroupValue<T, V> {
+    fn set_base_values(
+        &mut self,
+        shifted_idx: ShiftedTxnIndex,
```

Comment:
What does shifted transaction index mean here? Why is transaction index shifted?

Reply:
we need to distinguish storage value from all other transactions.

So shifted means 0 is storage, and i>0 means transaction i-1

I forgot the reason why we didn't do Option<TxnIndex> instead though (I mentioned but @gelash had a reason)